exampleCourse/questions/template/figure/dynamic/server.py

```diff
@@ -0,0 +1,57 @@
+import io
+import random
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def generate(data):
+    # Select a random shape.
+    shape = random.choice(["diamond", "circle"])
+    data["params"]["shape"] = shape
+
+    # Select a random radius.
+    radius = random.choice([1, 2, 3])
+    data["params"]["radius"] = radius
+
+    # Provide a label for the dimension.
+    if shape == "diamond":
```

Comment:
Nit: Below, you check for `shape == "circle"` but the opposite here. Making the conditions equal makes the question code easier to read.

Reply:
Done in 66e121f.